src/Apps/CollectorProfile/Routes/Saves2/Components/SavesItem.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, DROP_SHADOW, Flex, Text } from "@artsy/palette"
+import { FourUpImageLayout } from "Apps/CollectorProfile/Routes/Saves2/Components/FourUpImageLayout"
+import { StackedImageLayout } from "Apps/CollectorProfile/Routes/Saves2/Components/StackedImageLayout"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { SavesItem_item$data } from "__generated__/SavesItem_item.graphql"
+
+interface SavesItemProps {
+  isSelected?: boolean
+  imagesLayout: "stacked" | "grid"
+  item: SavesItem_item$data
+  onClick: () => void
+}
+
+const SavesItem: FC<SavesItemProps> = ({
+  isSelected,
+  imagesLayout,
+  item,
+  onClick,
+}) => {
+  const artworkNodes = extractNodes(item.artworksConnection)
+  const imageURLs = artworkNodes.map(node => node.image?.url ?? null)
+
+  return (
+    <Flex
```

Comment:
**question (non-blocking):** Thoughts on keyboard accessibility for these items? (Or does it make more sense to think about that once we actually make these navigable...)

Reply:
I will add support for this in another PR